release-team/role-handbooks/docs/README.md

```diff
@@ -55,7 +59,7 @@ In addition to the time requirements above, a Docs Lead must:
 Docs Lead Shadows are people who are preparing to be a Docs Lead in the future. In addition to the time requirements above, shadows must: 
 
 - Have signed the [contributor CLA](https://github.com/kubernetes/community/blob/master/CLA.md) for Kubernetes.
-- Be a Kubernetes org member. The process to become one is in the [Kubernetes community membership ladder](https://github.com/kubernetes/community/blob/master/community-membership.md#member)
+- Be a Kubernetes org member, or in the process of becoming an org member. The process to become one is in the [Kubernetes community membership ladder](https://github.com/kubernetes/community/blob/master/community-membership.md#member). This can often be achieved during the release cycle with sponsorship from a role lead.
```

Comment:
```suggestion
- Be invested in becoming an org member within the release cycle. This can often be achieved during the release cycle with sponsorship from a role lead. See the [Release Team onboarding guide](/release-team/release-team-onboarding.md) for more details.
```